site/src/components/AuditLogRow/AuditLogDiff.tsx

```diff
@@ -0,0 +1,109 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { AuditLog } from "api/typesGenerated"
+import { colors } from "theme/colors"
+import { combineClasses } from "util/combineClasses"
+
+const getDiffValue = (value: number | string | boolean) => {
+  if (typeof value === "string") {
+    return `"${value}"`
+  }
+
+  return value.toString()
+}
+
+export const AuditDiff: React.FC<{ diff: AuditLog["diff"] }> = ({ diff }) => {
```

Comment:
Looks like the file is AuditLogDiff but the component is AuditDiff, could you make them the same?

Reply:
Sure! I missed this.